src/Resources/skeleton/authenticator/EmptyAuthenticator.tpl.php

```diff
@@ -36,7 +36,7 @@ public function onAuthenticationFailure(Request $request, AuthenticationExceptio
         // todo
     }
 
-    public function onAuthenticationSuccess(Request $request, TokenInterface $token, $providerKey)
+    public function onAuthenticationSuccess(Request $request, TokenInterface $token, <?= $provider_key_type_hint ?> $providerKey)
```

Comment:
It's minor but ... will this generate an extra white space when `provider_key_type_hint` is empty? If that's the case, we may do this:

```diff
-TokenInterface $token, <?= $provider_key_type_hint ?> $providerKey)
+TokenInterface $token, <?= $provider_key_type_hint.' ' ?>$providerKey)
```

Reply:
Your right, it will generate an extra white space. Your suggestion also generate an extra white space.

Reply:
@javiereguiluz I made a changes to address this